packages/components/navigation/components/SideNotch.tsx

```diff
@@ -1,26 +1,33 @@
 import React from "react";
-import { StyleSheet, View, ViewStyle } from "react-native";
+import { View, ViewStyle } from "react-native";
 
 import SideNotchSVG from "../../../../assets/sidebar/side-notch.svg";
```

Comment:
should we use alias import to avoid long ../ :D

Reply:
We can use `@/` instead of `../../../..` yes.
I didn't change it here to limit the diffs